_posts/current/news/2020-05-13-96boards-promotion-in-collaboration-with-arrow-electronics-shiratech-and-st.md

```diff
@@ -0,0 +1,34 @@
+---
+layout: post
+title: '96Boards Promotion in collaboration with Arrow Electronics, Shiratech and ST'
+date: '2020-05-13 02:15:57'
+image: /assets/images/content/96boards-logo.png
+tags:
+  - 96Boards
+  - Arrow Electronics
+  - Shiratech
+  - ST
+  - Avenger96
+  - Promotion
+  - Giveaway
+  - Stinger96
+category: News
+author: jon.burcham@linaro.org
+---
+[96Boards](https://www.96boards.org/) have teamed up with[ Arrow Electronics](https://www.arrow.com/), [ST](https://www.st.com/content/st_com/en.html) and [Shiratech](https://www.shiratech-solutions.com/96boards-2/) and are excited to announce that there are 500 Avenger96 or Stinger96 boards to be given away.* Linaro are proud to enable the community to come together and to use these boards to build some new and exciting projects. Yang Zhang (Director at 96Boards) commented "We are excited to partner with 96Boards members and partners - Arrow, ST and Shiratech to launch this Campaign, giving away some of the most popular 96Boards compliant development platform featuring STM32MP157. This continues to show our commitment to the wide ecosystem, enabling developers from kernel all the way up to product designers; accelerating time to market with good upstreaming practice; serving SoC vendors and product development with dedicated and unified hardware and software support."
+
+The Avenger96 board featuring STM32MP157 is a highly integrated multi-market system-on-chip designed to enable secure and space constraint applications within the Internet of Things (IoT). The Avenger96 board features dual Arm Cortex-A7 cores and an Arm Cortex-M4 core. In addition, an extensive set of interfaces and connectivity peripherals are included to interface to cameras, touch-screen displays and MMC/SD cards. It also fully supports wireless communication, including WLAN and BLE. 
+
+![](/assets/images/content/96boards-avenger.png "96Boards Avenger")
```

Comment:
Can we use the image.html include here? This will ensure responsive images are generated.
E.g
{% include image.html path="/assets/images/content/96boards-avenger.png" alt="96Boards Avenger96 Board" %}